src/components/addbook.js

```diff
@@ -0,0 +1,29 @@
+import { useDispatch } from 'react-redux';
+import { v4 as uuidv4 } from 'uuid';
+import { addBook } from '../redux/books/books';
+
+const AddBook = () => {
+  const dispatch = useDispatch();
+
+  const submitBookToStore = () => {
+    const newBook = {
+      id: uuidv4(),
+      title: 'I need API. I am ready for data.',
+      author: 'human',
+    };
+
+    dispatch(addBook(newBook));
+  };
+
+  return (
+    <>
+      <h2>Add New Book</h2>
+      <form action="#">
+        <input type="text" required />
+        <button type="submit" onClick={submitBookToStore}>Add Book</button>
+      </form>
+    </>
+  );
```

Comment:
- [ ] The objective of the redux store is to take the data in input and add it to the store, right now I can only see the `submitBookToStore => object` are added.

Reply:
Hello @dasileker,
Thanks for the review. I can see what you pointed out and I have adjusted my code. But from what I can understand from the instructions this isn't a requirement at this stage.